src/plugins/http/src/httpResp.c

```diff
@@ -156,7 +156,10 @@ void httpSendErrorResp(HttpContext *pContext, int32_t errNo) {
 
   HttpServer *pServer = &tsHttpServer;
   SMonHttpStatus *httpStatus = monGetHttpStatusHashTableEntry(httpCode);
-  pServer->statusCodeErrs[httpStatus->index] += 1;
+  // FIXME(@huolinhe): I don't known why the errors index is overflowed, but fix it by index check
```

Comment:
no need to fix in develop branch?

Reply:
needed. in pr https://github.com/taosdata/TDengine/pull/9522